lec16-data-viz.Rmd

```diff
@@ -1,4 +1,97 @@
 ---
-title: "More on data visualizations (plus project work)"
+title: "More on data visualizations and the git workflow (plus project work)"
+author: Joel Ostblom
 ---
 
+First, let's create a chat room for your repository. We will use gitter, since
+it has good integrations with GitHub. Got to http://gitter.im/, login with your
+GitHub account and click the plus sign to create a new chat room. Browse to
+your repository, and add the members of your team. Since your repository is
+public, this chatroom will also be public (so you might not want to share
+personal information, and keep your discussions on topic, just as if this was
+a real-world work project).
```

Comment:
they won't need to do this. I'll be adding a README badge with a link to the gitter channel for them.

Reply:
I was thinking that it's good if they create the room, just so that they know how, and after that you will be adding the shield to the README.

Reply:
Sure, that makes sense